### PR TITLE
[DC-1309] Add Check for Blank Lines

### PR DESCRIPTION
**Pull request by jp3477**
This change checks for blank lines (i.e. comma separated nulls) and recommends submitting only the header.

### CODE REVIEW

File: omop_file_validator.py [hunk 1]
```diff
@@ -160,6 +160,23 @@ def find_error_in_file(column_name, cdm_column_type, submission_column_type,
...
+    :return: True if any rows has all empty values, else False
+    :rtype: bool
+    """
+    df = pd.read_csv(f)
+    if any(
+            df.apply(lambda row: all(row.apply(lambda col: pd.isnull(col))),
+                     axis=1)):
+        return True
+    else:
+        return False
```
Comment (nishanthpp93) [inline omop_file_validator.py]: nit: You can directly return the bool.
```suggestion
    return any(
            df.apply(lambda row: all(row.apply(lambda col: pd.isnull(col))),
                     axis=1))
```

File: omop_file_validator.py [hunk 2]
```diff
@@ -255,6 +272,15 @@ def run_checks(file_path, f):
         ]
         f.seek(0)
 
+        blank_lines = has_blank_lines(f)
+        if blank_lines:
+            blank_lines_msg = 'File contains blank lines.' \
+                              'If there is no data, please only submit the header line.'
+            result['errors'].append(dict(message=blank_lines_msg))
```
Comment (nishanthpp93) [inline omop_file_validator.py:281]: It would be helpful to see the row numbers for the lines here.

**Review by nishanthpp93 — CHANGES_REQUESTED** [review]
Added a suggestion and a minor comment. Also, you might want to run the script on the `examples_erroneous` directory and update the results file as more test cases are added.

File: omop_file_validator.py (hunk 2 above)
Comment (jp3477) [reply omop_file_validator.py:281]: Good idea, I'll add them.

File: omop_file_validator.py (hunk 1 above)
Comment (jp3477) [reply omop_file_validator.py]: Will make that change.

**Review by nishanthpp93 — COMMENTED** [review]
One last change: Try to run the script on the `examples_erroneous` directory and update the results files with the latest test cases.

**Review by nishanthpp93 — APPROVED** [review]
Looks great! Thanks for making the changes

**Pull request merged**